Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1654,6 +1654,19 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             FALSE,
             readWriteConstructParamFlags));
 
+    /**
+     * WebKitSettings:allow-scripts-to-close-windows:
+     *
+     * Allow scripts to close windows they did not open
+     */
+    g_object_class_install_property(gObjectClass,
+        PROP_ALLOW_DISPLAY_OF_INSECURE_CONTENT,
```

Comment:
Leftover copy. Will change to an actual `PROP_` value for this property and add missing switches to get/set value.